console/client/src/features/verbs/VerbCalls.tsx

```diff
@@ -14,10 +14,10 @@ type Props = {
 
 export const VerbCalls: React.FC<Props> = ({ module, verb }) => {
   const client = useClient(ConsoleService)
-  const [ calls, setCalls ] = useState<Call[]>([])
-  const { openPanel } = useContext(SidePanelContext)
+  const [ calls, setCalls ] = React.useState<Call[]>([])
+  const { openPanel } = React.useContext(SidePanelContext)
 
-  useEffect(() => {
+  React.useEffect(() => {
```

Comment:
What's the reason for this type of change? Do we want to prefer `React.<thing>` style vs just importing what we want? I probably have a preference to just use `useState` here vs. `React.useState` whatcha think?